index.js

```diff
@@ -156,7 +156,22 @@ Archive.prototype.download = function (i, cb) {
     if (err) return cb(err)
 
     var feed = self._getFeed(entry)
-    if (!feed) return cb(null)
+    var dest = join(self.directory, entry.name)
+
+    if (!feed) return createEmtpyEntry()
+
+    function createEmtpyEntry () {
```

Comment:
small nit pick. could you move this to the btm of the function with the other funcs?